examples/example-cfep-3.yml

```diff
@@ -1,5 +1,5 @@
 action: cfep3_copy
-anaconda_org_packages: # list of owner[@label=main]/pkg-name/version/subdir/filename.extension
- - jaimergp/conda-standalone/24.7.1/osx-arm64/conda-standalone-24.7.1-hce30654_0.conda
-from_anaconda_org_label: main # optional, default=main
+anaconda_org_packages:
+  # dict of (sha256, channel[/label/<label>]/subdir/filename.extension)
+  cd11f1f0fbe8c203212d002ca4e01795d0b1c886ce47d8b08e6bca3366a7c8e6: jaimergp/osx-arm64/conda-standalone-24.7.1-hce30654_0.conda
```

Comment:
Some how the ordering here feels backwards. Maybe map package name to sha, or does that have other issues?

Reply:
The sha256 is guaranteed to not contain a `:`, but I'm not sure if the label can have one. If that's the case, then the user would be forced to quote the key to prevent YAML from considering the `:` as a key, value separator. But tbh it's such an edge case that it doesn't really matter.
We can also do something like:

```yaml
- package: jaimergp/label/conda-standalone-24.7.1/osx-arm64/conda-standalone-24.7.1-hce30654_0.conda
  sha256: cd11f1f0fbe8c203212d002ca4e01795d0b1c886ce47d8b08e6bca3366a7c8e6
```

if you prefer it.

Reply:
Ahhhhh that makes sense. In yaml you can add quotes around the key for these edge cases. 

I like the list of entries better so let's go with that.

Reply:
Done!